### DISCUSSION TITLE
[Feature] Slideshow for albums

### DISCUSSION THREAD

**Discussion by comfreak89** [immich-app/immich#3881] [Feature Request]
### The feature

Now, that the slideshow feature has found its way to Immich, it would be awsome if it would be possible to do a slideshow on an album. For me it would be the perfect feature because I would use it as digital photoframe in the living room!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

Edit: Sorry, i did not use the search feature. Blame on me.

[https://github.com/immich-app/immich/discussions/3571](https://github.com/immich-app/immich/discussions/3571)
[https://github.com/immich-app/immich/discussions/2767](https://github.com/immich-app/immich/discussions/2767)

**Comment by alextran1502** [comment]
You can already use slideshow mode in an album

**Comment by comfreak89** [comment]
Uhh...sorry, but how? :/ 

I am on v1.75.1.

![image](https://github.com/immich-app/immich/assets/30108673/ad514c96-8223-44d2-bc56-cfa82a8a6dbb)

**Reply by jrasm91** [reply]
Go to the asset viewer first, by clicking on the first asset

**Comment by comfreak89** [comment]
Got it, my bad...

open a photo in the album, the 3 dots => slideshow.

/closed

**Comment by jrasm91** [comment]
To be fair, we could probably add the same menu item to the album menu.
